packages/content-management/table_list/src/services.tsx

```diff
@@ -104,7 +104,7 @@ export interface TableListViewKibanaDependencies {
           object: {
             references: SavedObjectsReference[];
           };
-          onClick?: (tag: { name: string }) => void;
+          onClick?: (tag: { name: string; description: string; color: string }) => void;
```

Comment:
@clintandrewhall That's one of those time consuming things with packages. I really only need the interface 

```typescript
{ name: string; }
```

But I am forced to add the rest of the props. After trying multiple things I give up and do what TS tells me :/

Glad to hear if I am missing something here though.